pkgs/tools/security/kwalletcli/default.nix

```diff
@@ -1,5 +1,14 @@
-{ mkDerivation, fetchFromGitHub, lib, makeWrapper, pkg-config
-, kcoreaddons, ki18n, kwallet, mksh, pinentry-qt }:
+{ mkDerivation
+, fetchFromGitHub
```

Comment:
This is not the correct formatting.

Reply:
Ah indeed, I accidentally used nixpkgs-fmt. I redid it with nixfmt.